src/EA.Weee.Web/Controllers/OrganisationRegistrationController.cs

```diff
@@ -881,5 +889,20 @@ public async Task<ActionResult> SoleTraderDetails(SoleTraderViewModel model)
 
             return RedirectToAction(nameof(OrganisationDetails), typeof(OrganisationRegistrationController).GetControllerName());
         }
+
+        private async Task ValidateProducerRegistrationNumber(string producerRegistrationNumber)
```

Comment:
@will-hope98  probably only want to do this when model.IsPreviousSchemeMember is true. Probably pass the model to the ValidateProducerRegistrationNumber method so you can do the entire flow here...